hax/helper/generate_sysconf.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python3
```

Comment:
Shebang is not needed - this is not an executable script per se.

Reply:
> Shebang is not needed - this is not an executable script per se.

Addressed.